### DISCUSSION TITLE
UCD Log - Content Design Guidance

### DISCUSSION THREAD

**Discussion by nhsbsa-deaco** [nhsbsa/nhsbsa-digital-playbook#250] [UCD]
As per other disciplines completing UCD log guidance, this discussion will be for guidance to support content designers in contributing to the UCD Log

**Comment by nhsbsa-briba** [comment]


**Reply by kimbell27** [reply]
This is a reply

**Comment by nhsbsa-debpo** [comment]
Just an idea but this might be a good place to keep our [Style Guide](https://nhsbsauk.sharepoint.com/sites/DigitalContentDesignTeam/SitePages/NHSBSA-digital-style-guide-and-standards.aspx). External users won't require site access to view and we'll be able to share it.

**Reply by kimbell27** [reply]
Yeah, that was one of the recommendations from the x-gov things wasn't it, to make the style guide public facing? Let's bring it up at the next CDC and we can have a chat about if we want to host or link to any resources on the playbook. I like [this idea from Interaction Design](https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/297) about having a section for new starters. If we do something like this and choose to link out to our CD resources from there, maybe that would be a place to look at hosting the style guide etc rather than the UCD design log? 😃

**Comment by allan-oconnor** [comment]
Here is the [Content Design UCD Log guidance doc](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/aloco_nhsbsa_nhs_uk/Ec9WPLQi3OVIkmLYgnM_iRUB7Q9ZRUu9qWyzICsur42dkQ?e=BvIcqd)

It is not finalised. Just uploading for info.

I am also attaching an [example of a CD UCD Log](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/aloco_nhsbsa_nhs_uk/EevLTtGUgClDpyoha0A2p0gBLvn6Q0tmNLK4vzi0RoGT8A?e=y0feGl), which is actually just a made up example.

**Comment by nhsbsa-deaco** [comment]
Finalised guidance for content design

[Content Design Guide for Design Logs.docx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14358460/Content.Design.Guide.for.Design.Logs.docx)

**Comment by nhsbsa-deaco** [comment]
Content Design Blank Template - Ready for Pilot

[Content Design UCD log template.docx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14534629/Content.Design.UCD.log.template.docx)
